cdp/network.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+from typing import Literal
+
+from cdp.client.models.network_identifier import NetworkIdentifier
+
+# Only SmartWallet related chains are listed here right now
+SupportedChainId = Literal[8453, 84532]
+
+_CHAIN_ID_TO_NETWORK_ID: dict[SupportedChainId, NetworkIdentifier] = {
+    8453: "base-mainnet",
+    84532: "base-sepolia",
+}
+
+
+@dataclass(frozen=True)
+class Network:
+    """Represents a network with its chain ID and network identifier."""
+
+    _chain_id: SupportedChainId
+    _network_id: NetworkIdentifier
```

Comment:
why `_` prefixes? I guess we aren't having users construct these but still is odd to me